app/components/EventDetails.css.ts

```diff
@@ -10,5 +10,9 @@ export const heading = style({
 });
 
 export const list = style({
-	margin: "1rem 0",
+	display: "flex",
+	flexDirection: "column",
+	gap: "0.75rem",
+	lineHeight: "2.75rem",
```

Comment:
The line height looks off on mobile. Maybe clamp the values.

Reply:
![screenshot of upcoming events styling on mobile](https://github.com/JoshuaKGoldberg/boston-ts-website/assets/114352576/33bfb7ee-6301-4478-9840-e68ba678bc5f)